cse-cc-export-runner-app/src/main/java/com/farao_community/farao/cse/export_runner/app/services/FileExporter.java

```diff
@@ -90,6 +104,35 @@ String saveRaoParameters(ProcessType processType) {
         return minioAdapter.generatePreSignedUrl(raoParametersDestinationPath);
     }
 
+    String saveTtcRao(CseRaoResult cseRaoResult, ProcessType processType, OffsetDateTime processTargetDate) {
+        StringWriter stringWriter = new StringWriter();
+        try {
+            JAXBContext jaxbContext = JAXBContext.newInstance(CseRaoResult.class);
+            Marshaller jaxbMarshaller = jaxbContext.createMarshaller();
+
+            // format the XML output
+            jaxbMarshaller.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, true);
+
+            QName qName = new QName("CseRaoResult");
+            JAXBElement<CseRaoResult> root = new JAXBElement<>(qName, CseRaoResult.class, cseRaoResult);
+
+            jaxbMarshaller.marshal(root, stringWriter);
+
+        } catch (JAXBException e) {
+            throw new CseInternalException("XSD matching error");
+        }
+        ByteArrayInputStream is = new ByteArrayInputStream(stringWriter.toString().getBytes());
+        String ttcPath =  getDestinationPath(processType, GridcapaFileGroup.OUTPUT) + getTtcRaoResultOutputFilename(processTargetDate);
+        minioAdapter.uploadOutput(ttcPath, is);
```

Comment:
public void uploadOutput(String path, InputStream inputStream, @Nullable String targetProcess, @Nullable String type, @Nullable String validityInterval) you should use this method instead